redis/gradle.properties

```diff
@@ -17,3 +17,5 @@
 #
 description=Redis adapter for Calcite
 artifact.name=Calcite Redis
+# JUnit4 use is allowed until the rest of the tests are migrated to JUnit5
+junit4=true
```

Comment:
Would you please migrate tests to JUnit5 instead?

Reply:
> Would you please migrate tests to JUnit5 instead?

well, Let me change it.